Floofbot/Handlers/CommandHandler.cs

```diff
@@ -63,6 +63,11 @@ private async Task HandleCommandAsync(SocketMessage s)
             {
                 var result = await _commands.ExecuteAsync(context, argPos, _services);
 
+                if (result.Error == CommandError.UnknownCommand && msg.HasMentionPrefix(_client.CurrentUser, ref argPos) && msg.ToString().EndsWith("?")) {
```

Comment:
I moved the test for question to the handler, since I think it should not make extra calls to modules that might not do anything.